### ISSUE TITLE
HTTPS console inaccessible via browser

### ISSUE THREAD

**Issue by anjackson** [internetarchive/heritrix3#279]
Odd issue here, in that when I run the latest `master`, I can't seem to login via the browser. It seems to work from the command-line using `curl`:

```
$ curl -v -k -u admin:admin https://localhost:8443/engine
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 8443 failed: Connection refused
*   Trying fe80::1...
* TCP_NODELAY set
* Connection failed
* connect to fe80::1 port 8443 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES128-GCM-SHA256
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: CN=Heritrix Ad-Hoc HTTPS Certificate
*  start date: Aug 11 23:05:01 2019 GMT
*  expire date: Aug  8 23:05:01 2029 GMT
*  issuer: CN=Heritrix Ad-Hoc HTTPS Certificate
*  SSL certificate verify result: self signed certificate (18), continuing anyway.
* Server auth using Basic with user 'admin'
> GET /engine HTTP/1.1
> Host: localhost:8443
> Authorization: Basic YWRtaW46YWRtaW4=
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Content-Type: application/xml; charset=UTF-8
< Date: Sun, 11 Aug 2019 23:08:11 GMT
< Accept-Ranges: bytes
< Server: Restlet-Framework/2.4.0
< Vary: Accept-Charset, Accept-Encoding, Accept-Language, Accept
< Transfer-Encoding: chunked
< 
<?xml version="1.0" standalone='yes'?>
<engine>
  <heritrixVersion>3.4.0-SNAPSHOT-2019-08-11T18:24:45Z</heritrixVersion>
  <heapReport>
    <usedBytes>13659864</usedBytes>
    <totalBytes>257425408</totalBytes>
    <maxBytes>954728448</maxBytes>
  </heapReport>
  <jobsDir>/Users/andy/Documents/workspace/heritrix3/dist/target/heritrix-3.4.0-SNAPSHOT/jobs</jobsDir>
  <jobsDirUrl>https://localhost:8443/engine/jobsdir/</jobsDirUrl>
  <availableActions>
    <value>rescan</value>
    <value>add</value>
    <value>create</value>
  </availableActions>
  <jobs></jobs>
</engine>

* Connection #0 to host localhost left intact
```

Firefox provides a login box but the credentials appear not to work. Chrome doesn't appear to offer a login at all.

Perhaps related to the Jetty upgrade?  #276

**Comment by ato** [comment]
Can reproduce. It seems to be accepting basic auth only - note that `curl --digest` is not working.
I think I missed this in my testing because the browser had cached basic auth credentials.

**Comment by ato** [comment]
Should be fixed by 36003b8626273fda6a643d719f2d973ef0393655. The authentication classes were restructured in Restlet 2 and I picked the wrong one. Sorry for the breakage.

**Comment by ato** [comment]
Actually make that 38c44e8e8e6189bdd46e7ffeb80fa7657eb27fd5. :man_facepalming:
